src/redemptions/redemption.service.ts

```diff
@@ -138,6 +150,59 @@ export class RedemptionService {
     return null;
   }
 
+  async repeatedFaceCheck(
+    response: JumioTransactionRetrieveResponse,
+  ): Promise<{ status: KycStatus; failureMessage: string } | null> {
+    // can only accept if no other errors occurred AND repeated face is from previous transaction of SAME user
+    let potentialFailures: string[] = [];
+    potentialFailures = potentialFailures.concat(
+      response.capabilities.dataChecks.map((i) => i.decision.type),
+      response.capabilities.extraction.map((i) => i.decision.type),
+      response.capabilities.liveness.map((i) => i.decision.type),
+      response.capabilities.similarity.map((i) => i.decision.type),
+      response.capabilities.usability.map((i) => i.decision.type),
+    );
+    // if any other check failed, we can't pass
+    for (const potentialFailure of potentialFailures) {
+      if (potentialFailure !== 'PASSED') {
+        return null;
+      }
+    }
+    // for imageChecks, only duplicate face should pass
+    const failedImageChecks = response.capabilities.imageChecks.filter(
+      (i) => i.decision.type !== 'PASSED',
```

Comment:
is there another status besided `PASSED`? Should we be precise with the type we know accompanies a duplicated face?

Reply:
decision.type is `WARNING` for `REPEATED_FACE` label, there is another label for warning  `DIFFERENT_PERSON`, so we should check them all (ie not all `WARNING`s are `REPEATED_FACE`